zeroize/build.rs

```diff
@@ -27,7 +27,9 @@ mod linux {
         let glibc_version = get_glibc_version();
 
         // Hax: this probably isn't the best use of floats
-        if glibc_version.parse::<f32>().unwrap() < GLIBC_WITH_EXPLICIT_BZERO.parse::<f32>().unwrap()
+        if glibc_version.is_none()
```

Comment:
I think this should be `is_some()`. Musl appears to support `explicit_bzero()` natively, and for now I think the shim should explicitly target legacy glibc versions (with the goal of eventually removing it).

Reply:
I get undefined reference against v1.1.9 of musl-libc. It should be `is_none()` because the version of glibc is not known and it's not glibc, the function returns None only when it couldn't find a version and in that case it should compile explicit_bzero. I hope I'm not missing a detail there.

Reply:
@tarcieri I've checked, `explicit_bzero` is a recent addition (only in 1.1.20, last September) to musl-libc. Should check for version and only compile the backport for <1.1.20? But then we'd need to add a function to tell which standard library is being used before looking at version numbers.

Reply:
In that case, perhaps add a separate `get_musl_version()` which also returns an `Option<String>`?

Reply:
> But then we'd need to add a function to tell which standard library is being used before looking at version numbers.

If you add a `get_musl_version()` with the same signature you can more or less do this:

```rust
if let Some(version) = get_glibc_version() {
   [...]
} else if let Some(version) = get_musl_version() {
  [...]
} else {
}
```

Reply:
done, see latest commits.